src/site/content/en/blog/webtransport/index.md

```diff
@@ -130,22 +142,20 @@ await transport.ready;
 
 Once you have a WebTransport instance that's connected to a server, you can use it to send and receive discrete bits of data, known as [datagrams](https://en.wikipedia.org/wiki/Datagram).
 
-The `sendDatagrams()` method returns a <code>[WritableStream](https://developer.mozilla.org/en-US/docs/Web/API/WritableStream)</code>, which a web client can use to send data to the server. The <code>receiveDatagrams()</code> method returns a <code>[ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream)</code>, allowing you to listen for data from the server. Both streams are inherently unreliable, so it is possible that the data you write will not be received by the server, and vice versa.
+The `writeable` getter returns a <code>[WritableStream](https://developer.mozilla.org/en-US/docs/Web/API/WritableStream)</code>, which a web client can use to send data to the server. The <code>readable</code> getter returns a <code>[ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream)</code>, allowing you to listen for data from the server. Both streams are inherently unreliable, so it is possible that the data you write will not be received by the server, and vice versa.
```

Comment:
I can't find `writeable` or `readable`. What objects do they come from?

Reply:
They are from `transport.datagrams` that is of *DatagramDuplexStream Interface*, but the current write-up looks as if they are directly on the `transport` object.

Reply:
The code sample in the following section should be accurate now, and show how to access `writeable` and `readable`.

Reply:
I don't know why I couldn't find those yesterday. I see them now.